x-pack/plugins/ui_actions_enhanced/public/drilldowns/hooks/i18n.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const toastDrilldownCreated = {
+  title: (drilldownName: string) =>
+    i18n.translate(
+      'xpack.uiActionsEnhanced.drilldowns.components.flyoutDrilldownWizard.toast.drilldownCreatedTitle',
+      {
+        defaultMessage: 'Drilldown "{drilldownName}" created',
+        values: {
+          drilldownName,
+        },
+      }
+    ),
+  text: i18n.translate(
+    'xpack.uiActionsEnhanced.drilldowns.components.flyoutDrilldownWizard.toast.drilldownCreatedText',
+    {
+      // TODO: remove `Save your dashboard before testing.` part
+      // when drilldowns are used not only in dashboard
+      // or after https://github.com/elastic/kibana/issues/65179 implemented
```

Comment:
Should we just remove it? I doubt that it brings any value and removing it will make drilldown more generic :)

Reply:
Yes, I agree on removing it, but let's ask Jason if he is OK with it, too, I've added it to the sync agenda.

Reply:
(From a meeting) "Save you changes before testing". We will remove after https://github.com/elastic/kibana/issues/65179 is done